frontend/src/component/menu/routes.ts

```diff
@@ -267,6 +267,15 @@ export const routes: IRoute[] = [
         component: EnvironmentTable,
         type: 'protected',
         flag: EEA,
+        notFlag: 'purchaseAdditionalEnvironments',
+        menu: { mobile: true, advanced: true },
+    },
+    {
+        path: '/environments',
```

Comment:
Will these both pop up for enterprise?

Reply:
No, because other "path" has `notFlag`. Only 1 item shows up on Enterprise - tested in sandbox.